src/main/java/com/bamdoliro/maru/application/form/UpdateSecondRoundScoreUseCase.java

```diff
@@ -51,79 +50,142 @@ public void execute(MultipartFile xlsx) throws IOException {
-                ((row.getCell(2).getStringCellValue().equals("마이스터인재전형") &&
-                        ((row.getCell(3).getNumericCellValue() > 120 || row.getCell(4).getNumericCellValue() > 40 || row.getCell(5).getNumericCellValue() > 80) ||
-                                (row.getCell(3).getNumericCellValue() < 0 || row.getCell(4).getNumericCellValue() < 0 || row.getCell(5).getNumericCellValue() < 0))) ||
-                        (row.getCell(2).getStringCellValue().equals("사회통합전형") &&
-                                ((row.getCell(3).getNumericCellValue() > 200 || row.getCell(4).getNumericCellValue() > 40) ||
-                                        (row.getCell(3).getNumericCellValue() < 0 || row.getCell(4).getNumericCellValue() < 0))) ||
-                        (!row.getCell(2).getStringCellValue().equals("마이스터인재전형") &&
-                                !row.getCell(2).getStringCellValue().equals("사회통합전형") &&
-                                ((row.getCell(3).getNumericCellValue() > 120 || row.getCell(4).getNumericCellValue() > 40) ||
-                                        (row.getCell(3).getNumericCellValue() < 0 || row.getCell(4).getNumericCellValue() < 0))))) {
-            throw new WrongScoreException();
+        if (isShow) {
+            if (depthInterviewScoreCell != null && depthInterviewScoreCell.getCellType() != CellType.NUMERIC) invalidCellTypeList.add(depthInterviewScoreCell);
+            if (ncsScoreCell != null && ncsScoreCell.getCellType() != CellType.NUMERIC) invalidCellTypeList.add(ncsScoreCell);
+            if (!(codingTestScoreCell == null || codingTestScoreCell.getCellType() == CellType.NUMERIC || codingTestScoreCell.getCellType() == CellType.BLANK)) {
+                invalidCellTypeList.add(codingTestScoreCell);
+            }
         }
+
+        return invalidCellTypeList;
+    }
+
+    private List<Cell> validateScore(Row row) {
+        // 수험번호 | 이름 | 전형 구분 | 심층면접 | NCS | 코딩테스트 | 응시 여부
+        Cell typeCell = row.getCell(2);
+        Cell depthInterviewScoreCell = row.getCell(3);
+        Cell ncsScoreCell = row.getCell(4);
+        Cell codingTestScoreCell = row.getCell(5) != null ? row.getCell(5) : null;
+        boolean isShow = row.getCell(6).getCellType() == CellType.FORMULA && row.getCell(6).getBooleanCellValue();
+
+        List<Cell> wrongScoreCellList = new ArrayList<>();
+
+        if (isShow) {
+            String type = typeCell.getStringCellValue();
+            double depthInterviewScore = depthInterviewScoreCell.getNumericCellValue();
+            double ncsScore = ncsScoreCell.getNumericCellValue();
+            double codingTestScore = codingTestScoreCell != null ? codingTestScoreCell.getNumericCellValue() : 0;
+
+            switch (type) {
+                case "마이스터인재전형" -> {
+                    if (depthInterviewScore < 0 || depthInterviewScore > 120)
+                        wrongScoreCellList.add(depthInterviewScoreCell);
+                    if (ncsScore < 0 || ncsScore > 40)
```

Comment:
소소한 팁인데
이런식으로 쓰면 가독성이 더 좋더라구욧

```suggestion
                    if (ncsScore < 0 || 40 < ncsScore)
```

```suggestion
                    if (!(0 <= ncsScore && ncsScore <= 40))
```

Reply:
> 소소한 팁인데 이런식으로 쓰면 가독성이 더 좋더라구욧

저는 if 문 안에 되도록이면 긍정문을 넣고, 중첩 괄호를 지양하려고 했었는데
두번째처럼 하는 것은 실제 수학식 `0 <= ncsScore <= 40` 처럼 보이게 하기 위해서인가요?

Reply:
네 두번째는 그 의도입니당